docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,560 @@
+
+Now in Flink 1.20, you can use the `DQL` syntax to obtain detailed metadata from existing catalogs, and the
+`DDL` syntax to modify metadata such as properties or comment in the specified catalog.
+
+```sql
+Flink SQL> CREATE CATALOG `cat` WITH ('type'='generic_in_memory', 'default-database'='db');
+[INFO] Execute statement succeeded.
+
+Flink SQL> SHOW CREATE CATALOG `cat`;
++---------------------------------------------------------------------------------------------+
+|                                                                                      result |
++---------------------------------------------------------------------------------------------+
+| CREATE CATALOG `cat` WITH (
+  'default-database' = 'db',
+  'type' = 'generic_in_memory'
+)
+|
++---------------------------------------------------------------------------------------------+
+1 row in set
+
+Flink SQL> DESCRIBE CATALOG `cat`;
++-----------+-------------------+
+| info name |        info value |
++-----------+-------------------+
+|      name |               cat |
+|      type | generic_in_memory |
+|   comment |                   |
++-----------+-------------------+
+3 rows in set
+
+Flink SQL> ALTER CATALOG `cat` SET ('default-database'='new-db');
+[INFO] Execute statement succeeded.
+
+Flink SQL> SHOW CREATE CATALOG `cat`;
++-------------------------------------------------------------------------------------------------+
+|                                                                                          result |
++-------------------------------------------------------------------------------------------------+
+| CREATE CATALOG `cat` WITH (
+  'default-database' = 'new-db',
+  'type' = 'generic_in_memory'
+)
+|
++-------------------------------------------------------------------------------------------------+
+1 row in set
+```
+
+**More Information**
+* [FLINK-34914](https://issues.apache.org/jira/browse/FLINK-34914)
+* [FLIP-436](https://cwiki.apache.org/confluence/display/FLINK/FLIP-436%3A+Introduce+Catalog-related+Syntax)
+
+
+## Add DISTRIBUTED BY Clause
+
+Many SQL engines expose the concepts of `Partitioning`, `Bucketing`, or `Clustering`. We propose to introduce
+the concept of `Bucketing` to Flink.
+
+Buckets enable load balancing in an external storage system by splitting data into disjoint subsets. It depends
+heavily on the semantics of the underlying connector. However, a user can influence the bucketing behavior by
+specifying the number of buckets, the bucketing algorithm, and (if the algorithm allows it) the columns which 
+are used for target bucket calculation. All bucketing components (i.e. bucket number, distribution algorithm, bucket key columns)
+are optional from a SQL syntax perspective.
```

Comment:
NIT: are optional from a SQL syntax perspective. - >are optional in the SQL syntax.